Assets/Scripts/GameUI.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
```

Comment:
A game over screen is nice but normally this is put in a separate scene, that way the island/player/etc isn't simulated in the background while the ending screen plays.